Gemfile

```diff
@@ -1,8 +1,8 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.0rc1'
+gem 'rails', '4.2'
 
-gem 'active_model_serializers'
+gem 'active_model_serializers', '~> 0.9.0'
```

Comment:
as far as I tested, it's working. it's probably because of the configs we have in the initializers. but I'll downgrade